checklists/lz_checklist.en.json

```diff
@@ -75,11 +75,11 @@
     {
       "category": "Identity and Access",
       "subcategory": "Identity",
-      "text": "If the following roles are considered, use Azure custom roles: Azure platform owner, network management, security operations, subscription owner, application owner",
+      "text": "Consider using Azure custom roles for the following key roles: Azure platform owner, network management, security operations, subscription owner, application owner",
       "guid": "f5664b5e-984a-4859-a773-e7d261623a76",
       "severity": "Medium",
       "training": "https://docs.microsoft.com/learn/modules/create-custom-azure-roles-with-rbac/",
-      "link": "https://docs.microsoft.com/azure/cloud-adoption-framework/ready/landing-zone/design-area/identity-access#prerequisites-for-a-landing-zone---design-recommendations"
+      "link": "https://docs.microsoft.com/azure/cloud-adoption-framework/ready/landing-zone/design-area/identity-access"

```

Comment:
Would you be able to update your branch to the previous link? https://docs.microsoft.com/azure/cloud-adoption-framework/ready/landing-zone/design-area/identity-access#prerequisites-for-a-landing-zone---design-recommendations

Reply:
Should be fixed now - thank you for spotting this.